Src/coffeescript/cql-execution/src/elm/clinical.coffee

```diff
@@ -11,7 +11,7 @@ module.exports.ValueSetDef = class ValueSetDef extends Expression
     #todo: code systems and versions
 
   exec: (ctx) ->
-    valueset = ctx.codeService.findValueSet(@id, @version) ? new ValueSet(@id, @version)
+    valueset = ctx._codeService.findValueSet(@id, @version) ? new ValueSet(@id, @version)
```

Comment:
It's odd that you needed to do that.  The `Context` class should be exposing a `codeService` getter so you don't have to access the private `_codeService` property directly.  See: https://github.com/cqframework/clinical_quality_language/blob/cccc2144b17c55d111631d9291c2e4e3f6b9b273/Src/coffeescript/cql-execution/src/runtime/context.coffee#L16-L18